met-web/src/components/engagement/view/widgets/PhasesWidget/EngagementPhase.tsx

```diff
@@ -27,12 +27,11 @@ export const EngagementPhase = ({
             sx={{ borderRight: isCurrent ? 'none' : `1px solid ${PAST_PHASE.borderColor}` }}
             learnMoreBox={
                 <ReadMoreBox
-                    backgroundColor={isCurrent ? learnMoreBackgroundColor : PAST_PHASE.backgroundColor}
+                    backgroundColor={isCurrent ? learnMoreBackgroundColor : PAST_PHASE.readmoreBackgroundColor}
```

Comment:
change readmoreBackgroundColor to readMoreBackgroundColor (make the m of more in caps)